monitoring/types.go

```diff
@@ -23,6 +24,7 @@ const (
 	instanceManagerLabel = "instance_manager"
 	instanceManagerType  = "instance_manager_type"
 	managerLabel         = "manager"
+	backupsLabel         = "backups"
```

Comment:
Suggest `backupLabel` instead.

Reply:
Fix in d156d1fdaa05700e384ff7ab0ea429f15ee27ab4